xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/hierarchy.vm

```diff
@@ -24,7 +24,7 @@
   #######################################################
   ##                   CONTROLLER
   ##
-  ## Call the appropiate breadcrumb depending on the 
+  ## Call the appropiate breadcrumb depending on the
```

Comment:
```suggestion
  ## Call the appropiate breadcrumb depending on the 
```

While the trailing space is wrong, it's the only change so shouldn't be committed here.

Reply:
... My IDE was auto removing the trailing line on save of edited lines...
Addressed in https://github.com/xwiki/xwiki-platform/pull/2133/commits/f969fc7ea1210f19c65693bfb0fbd9fdb635097a :+1: